docs/pages/reference/yaml-reference.md

```diff
@@ -32,6 +32,10 @@ This documentation effectively reflects the Java object tree under [RawServiceSp
 
     Custom zookeeper URL for storing scheduler state. Defaults to `master.mesos:2181`.
 
+  * `user`
+
+    The system user to run the service's scheduler and pods as. Availability of usernames depends on the cluster. If DC/OS Security is enabled, this may need to be set to `nobody`, or the administrator may be required to configure custom permissions allowing the service to run as a different user.
```

Comment:
I thought the scheduler runs as `root`?

Reply:
No it runs as the user specified here -- i.e. as the same user the service itself runs as. (had checked this via the code)

Reply:
That's theoretically true with the caveat that the user specified in the Marathon app definition much be the same as the one set in the YAML.

Reply:
The second sentence should read:

"In DC/OS Enterprise, if the security mode is set to `strict`, the service account used by the service must have permissions to launch tasks as the assigned user."

Also, is there a default? Or is it a required field?